juriscraper/pacer/email.py

```diff
@@ -202,6 +204,40 @@ def _get_description_plain(self):
             return clean_string(description)
         return None
 
+    def _contains_attachments(self) -> bool:
+        """Determines if the html notification contains attached documents.
+
+        :returns: True if it contains otherwise False.
+        """
+
+        path = '//p[contains(., "The following document(s) are associated with this transaction:")]/following::'
+        document_nodes = self.tree.xpath(
+            f'{path}strong[contains(., "Document description:")]'
+        )
```

Comment:
Why break up the path here? I'd suggest reformatting it and adding whatever whitespace you need by putting it in `"""`-blocks. Something like:

```
"""
//p[contains(., "The following document(s) are associated with this transaction:")]
  /following::strong[contains(., "Document description:")]
"""
```

Reply:
Also, isn't the second line enough? 

```
path = //strong[contains(., "Document description:")]
```

(I'm just guessing, not really playing with it.)

Reply:
Yeah, this last one `path = //strong[contains(., "Document description:")]` works properly, thanks.